.circleci/config.yml

```diff
@@ -318,32 +324,4 @@ workflows:
             branches:
               ignore: /.*/
             tags:
-              only: /master-patch.*/
-
-      - orb-tools/dev-promote-prod:
-          name: dev-promote-minor
-          release: minor
-          context: orb-publishing
-          orb-name: circleci/orb-tools
-          ssh-fingerprints: 33:d9:6e:a7:b2:ba:eb:90:7b:7a:dc:90:36:e1:b6:e0
-          cleanup-tags: true
-          requires: *prod-deploy_requires
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /master-minor.*/
-
-      - orb-tools/dev-promote-prod:
-          name: dev-promote-major
-          release: major
-          context: orb-publishing
-          orb-name: circleci/orb-tools
-          ssh-fingerprints: 33:d9:6e:a7:b2:ba:eb:90:7b:7a:dc:90:36:e1:b6:e0
-          cleanup-tags: true
-          requires: *prod-deploy_requires
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /master-major.*/
+              only: /master-.*/
```

Comment:
Do we still want this a tag-only operation? The new labeling allows us to run it directly on master.

Reply:
@eddiewebb Currently the config has 2 workflows; one workflow to do linting, validation and publishing of the dev orb, which then kicks off the 2nd workflow by pushing git tags. If we change the filters for this job to run on `master` branch I believe it would mean that the 2 workflows would run in parallel.

I think it'd be good to use the v2 API instead of git tags to trigger the 2nd workflow, but I think that might be better for a separate PR to keep this PR smaller?

Reply:
I was just looking into creating that PR. +1 to separate PR, but we could put this into a staging branch maybe? We could get this to a good spot and do a major update.

Reply:
@KyleTryon Sure, I'll put this in a staging branch!